sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParserTest.java

```diff
@@ -134,12 +134,61 @@ public void branchCoverage() throws Exception {
     assertThat(coverage.getBranchCoverage(filePath).get(0)).isEqualTo(new BranchCoverage(9, 4 , 2));
     assertThat(coverage.getBranchCoverage(filePath).get(1)).isEqualTo(new BranchCoverage(12, 2 , 1));
     assertThat(coverage.getBranchCoverage(filePath).get(2)).isEqualTo(new BranchCoverage(13, 2, 1));
-    assertThat(coverage.getBranchCoverage(filePath).get(3)).isEqualTo(new BranchCoverage(14, 4, 2));
+    assertThat(coverage.getBranchCoverage(filePath).get(3)).isEqualTo(new BranchCoverage(14, 6, 4));
     assertThat(coverage.getBranchCoverage(filePath).get(4)).isEqualTo(new BranchCoverage(18, 6, 3));
   }
 
+    @Test
+  public void branchCoverage_getter_setter_complex_case() throws Exception {
```

Comment:
We could use overlapping_segments instead of complex, WDYT?

Reply:
I will use "multiple_sequence_points_per_line" because the sequence points are not overlapping (they are on separate positions)